presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -382,3 +382,119 @@ node.
 
 You should ensure that the keytab files have the correct permissions on every
 node after distributing them.
+
+
+Hive Data Stored in Amazon S3
+=============================
+
+The Hive Connector can read data from files stored in S3, assuming the Hive
+Metastore has tables defined as "external" which have locations defined as
+S3 URIs.
+
+Presto registers its own S3 filesystem for the following URI schemes:
+``s3://``, ``s3a://``, ``s3n://`` and it registers the default Hadoop S3
+FileSystem (``org.apache.hadoop.fs.s3.S3FileSystem``) for ``s3bfs://`` URIs.
+The following assumes you are using the native Presto S3 FileSystem implementation.
+
+Configuration Properties
```

Comment:
Yeah, I actually think this should just be in a separate document, hive-s3.rst or something.  I feel like covering all the AWS/S3 related items in the same document makes sense.

Thoughts?